src/Api/UserApi.php

```diff
@@ -124,6 +124,10 @@ public function validate(Register $register): ValidationSchema
     {
       $validation_schema->setUsername($this->translator->trans('api.registerUser.usernameAlreadyInUse', [], 'catroweb'));
     }
+    elseif (0 === strncmp($register->getUsername(), User::$SCRATCH_PREFIX, strlen(User::$SCRATCH_PREFIX)))
```

Comment:
AH nice, you already thought about this! :)
So, we just need to update the CAPI documentation